mips-lib/src/main.rs

```diff
@@ -0,0 +1,31 @@
+use clap::Parser;
+use mips_lib;
+use std::path::PathBuf;
+#[cfg(feature = "gui-egui")]
+use syncrim::gui_egui::editor::Library;
+use syncrim::{common::ComponentStore, fern::fern_setup};
+/// Simple program to greet a person
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Args {
+    /// Path to the model to load on startup
+    #[arg(short, long, default_value = "empty.json")]
```

Comment:
maybe change this to the single cycle mips?

Reply:
fixed in d19ff005c6cada6c3a12b9012063ec30e0cd3c5c